ksp_plugin/vessel.cpp

```diff
@@ -116,6 +116,9 @@ void Vessel::CreateHistoryAndForkProlongation(
   CHECK(!is_initialized());
   history_ = std::make_unique<DiscreteTrajectory<Barycentric>>();
   history_->Append(time, degrees_of_freedom);
+  // TODO(egg): proper initialization.
+  psychohistory_.Append(time, degrees_of_freedom);
+  psychohistory_is_history_ = true;
```

Comment:
I don't understand this.  Shouldn't it be `false` for compatibility?  Same question at line 229.